packages/core/src/crypto/KeyBackend.ts

```diff
@@ -0,0 +1,4 @@
+export enum KeyBackend {
+  Software = 'sofware',
+  SecureElement = 'secure_element',
+}
```

Comment:
```suggestion
export enum KeyBackend {
  Software = 'sofware',
  /**
  * Generate the key in a secure element. Need to be supported by the Wallet implementation. If using Aries Askar, Secure Element is available on iOS and Android in React Native
  */
  SecureElement = 'secure_element',
}
```

Reply:
I redefined the enum in credo so it would not be askar specific. I will add a comment to the KeyBackend enum, but I will slightly reword it.

Reply:
But the names `sofware` and `secure_element' are hardcoded in Askar, right? So in some way it depends on Askar (otherwise it would not be snake_case :smile: ).

Note the typo in 'sofware' (missing t).

Reply:
`software` and `secure_element` are also used by askar as it would not make sense to choose something else. If we want to support other providers which define different identifiers for their backend we can create a mapping function.

For now, this is fine IMO. (good catch as well on the `sofware` ;) ).

Reply:
> as it would not make sense to choose something else

It would, as Credo API != Askar API